core-bundle/src/Resources/contao/classes/BackendTemplate.php

```diff
@@ -180,7 +180,7 @@ protected function getLocaleString()
 			'var Contao={'
 				. 'theme:"' . Backend::getTheme() . '",'
 				. 'lang:{'
-					. 'close:"' . $GLOBALS['TL_LANG']['MSC']['close'] . '",'
```

Comment:
I would keep both here. I'm sure someone else is using the close label.

Reply:
I am not a big fan of sending data that is nowhere used, but for the sake of BC, I have re-added the label in 9296b6640ade7e23768e6f9008ff67123dfeb42d.